evm/src/WormholeTransceiver.sol

```diff
@@ -5,58 +5,58 @@ import "wormhole-solidity-sdk/WormholeRelayerSDK.sol";
 import "wormhole-solidity-sdk/libraries/BytesParsing.sol";
 import "wormhole-solidity-sdk/interfaces/IWormhole.sol";
 
-import "./libraries/EndpointHelpers.sol";
-import "./libraries/EndpointStructs.sol";
-import "./interfaces/IWormholeEndpoint.sol";
+import "./libraries/TransceiverHelpers.sol";
+import "./libraries/TransceiverStructs.sol";
+import "./interfaces/IWormholeTransceiver.sol";
 import "./interfaces/ISpecialRelayer.sol";
-import "./interfaces/IManager.sol";
-import "./Endpoint.sol";
+import "./interfaces/INttManager.sol";
+import "./Transceiver.sol";
 
-contract WormholeEndpoint is Endpoint, IWormholeEndpoint, IWormholeReceiver {
+contract WormholeTransceiver is Transceiver, IWormholeTransceiver, IWormholeReceiver {
     using BytesParsing for bytes;
 
     uint256 public constant GAS_LIMIT = 500000;
     uint8 public immutable consistencyLevel;
 
-    /// @dev Prefix for all EndpointMessage payloads
+    /// @dev Prefix for all TransceiverMessage payloads
     ///      This is 0x99'E''W''H'
```

Comment:
More intrusive, but should we change the payload prefix to match the naming change?

Reply:
Same goes for all these constants actually, the ones below aren't correct anymore (i.e. the comment does not align with the actual bytes)

Reply:
good point, let me fix those -- tempted to keep the 'E' 'W' 'H' though. what do you think?

Reply:
Yeah that one is maybe worth keeping as is